pyccel/ast/builtin_methods/set_methods.py

```diff
@@ -147,6 +148,34 @@ def __init__(self, set_variable):
         self._class_type = set_variable.class_type.element_type
         super().__init__(set_variable)
 
+#==============================================================================
+class SetRemove(SetMethod):
+    """
+    Represents a call to the .remove() method.
+
```

Comment:
Bad merge? This looks like you reverted #2034

Reply:
The test was failing. I had to search and add it. It was present in the "devel" branch.

Reply:
Which test was failing? This class used to be present in the `devel` branch but it was removed in  #2034. As you can see on the PR, the tests were passing after its removal